report_qweb_signer/models/report.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+from contextlib import closing
+import os
+import subprocess
+import tempfile
+import time
+
+from openerp import SUPERUSER_ID, models, api, _
+from openerp.exceptions import Warning, AccessError
+from openerp.tools.safe_eval import safe_eval as eval
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+def _normalize_filepath(path):
+    path = path or ''
+    path = path.strip()
+    if not path.startswith('/'):
+        me = os.path.dirname(__file__)
+        path = '{}/../static/certificate/'.format(me) + path
+    if not os.path.exists(path):
+        return False
+    return path
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    def _user_company_id_get(self, cr, uid, context=None):
+        m_users = self.pool.get('res.users')
+        user = m_users.browse(cr, SUPERUSER_ID, uid, context=context)
+        return user.company_id.id
```

Comment:
I write in old API because I have to inherit from ```get_pdf```, called from a controller and in v7 form.
I got issues when start in new API and then I go with old one. But maybe we can migrate before merge.
Hown can inherit from ```get_pdf``` method at ```report``` core addon, using new API?